src/.defaultsForValidator.js

```diff
@@ -136,7 +135,8 @@ const defaults = {
 */
 const deprecated = {
   'no_produces_for_get': 'no_produces',
-  'parameters.snake_case_only': 'param_name_case_convention'
+  'parameters.snake_case_only': 'param_name_case_convention',
+  'undefined_required_properties': 'missing-required-property',
```

Comment:
I forget how this value gets used (I think in a warning message) but it may be good to call out the fact that it's a spectral rule

Reply:
Good point! The key is the old rule, and the value is the replacement rule if it exists.

I could do something simple like use `missing-required-property (Spectral rule)`, which will result in, "Use `missing-required-property (Spectral rule)` instead."

That might be confusing, though, so I could update the deprecation code to produce a different message when the new rule is a Spectral rule.

Reply:
I think that's fine, just something to indicate that the rule isn't going to be found in the validaterc file like the old one

Reply:
added the note in parentheses